assets/completions/bat.zsh

```diff
@@ -0,0 +1,56 @@
+#compdef bat
+# FIXME: help me with the subcommand `cache`, zsh completion is hard as hell
```

Comment:
I have never written any completion scripts. Let's leave this open until someone can help?

Reply:
Maybe I'll dive into the zsh manual again later, sorry to say but is one of the most difficult manual I have read IMO 🤣

Reply:
I'd be also okay with merging this without support for the `cache` subcommand.

Reply:
I've added the completion of `cache` subcommand in https://github.com/sharkdp/bat/pull/1136/commits/32905f78dd60a2a181b3792d73ba8e5701736ddb